modules/bootconf.py

```diff
@@ -118,3 +127,94 @@ def configure(self, old, new):
             return False
 
         return True
+
+class UBootBeagleboneBootloader(BootloaderConfigurator):
+
+    def switchUEnv(self, old, new, uEnvPath):
+        ''' Switch boot partition using uEnv bootpart variable '''
```

Comment:
Hmm... the comment is misleading. It actually was meant to sounds as "switch the partition from which to boot - aka root partition). I will change to clear this up.